UnityProject/Assets/Scripts/UI/Core/ChangeLogs/ChangeLogEntryUI.cs

```diff
@@ -2,17 +2,43 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
+using TMPro;
 
 namespace UI
 {
 	public class ChangeLogEntryUI : MonoBehaviour
 	{
-		public Text text;
+		public TMP_Text text;
+
+		public static class ChangelogIcons
+		{
+			public static string FIX => "";
+			public static string BALANCE => "";
+			public static string NEW => "";
+			public static string IMPROVE => "";
+			public static string UNKNOWN => "?";
+		}
 
 		public void SetEntry(ChangeLogEntry entryData)
 		{
-			text.text = $"---------------------------------------------------\n\nDate: {entryData.date} \n\rAuthor: {entryData.author} " +
-			 $"\n\r\n\rCommit: {entryData.commit} \n\r\n\rMessage: {entryData.message}";
+			text.text = $"{CatagoryGrabber(entryData.category)} - {entryData.description}\n\rAuthor: {entryData.author_username}";
```

Comment:
why not link to the pr and author?

Reply:
was still using text at that time, not tmp

Reply:
aight, leave it as is. I will make a pr to your branch with some changes